soc_043_rw0_refugees_internally_displaced_persons/soc_043_rw0_refugees_internally_displaced_persons_processing.py

```diff
@@ -0,0 +1,176 @@
+import logging
+import pandas as pd
+import glob
+import os
+import sys
+import dotenv
+#insert the location of your .env file here:
+utils_path = os.path.join(os.path.abspath(os.getenv('PROCESSING_DIR')),'utils')
+if utils_path not in sys.path:
+    sys.path.append(utils_path)
+import util_files
+import util_cloud
+import util_carto
+import zipfile
+from zipfile import ZipFile
+import shutil
+
+# Set up logging
+# Get the top-level logger object
+logger = logging.getLogger()
+for handler in logger.handlers: logger.removeHandler(handler)
+logger.setLevel(logging.INFO)
+# make it print to the console.
+console = logging.StreamHandler()
+logger.addHandler(console)
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+# name of table on Carto where you want to upload data
+# using preexisting table for this dataset
+dataset_name = 'soc_043_rw0_refugees_internally_displaced_persons' #check
+
+logger.info('Executing script for dataset: ' + dataset_name)
+# create a new sub-directory within your specified dir called 'data'
+# within this directory, create files to store raw and processed data
+data_dir = util_files.prep_dirs(dataset_name)
+
+'''
+Download data and save to your data directory
+Data can be downloaded at the following link:
+https://www.unhcr.org/refugee-statistics/download
+
+One csv file was downloaded from the explorer
+after selecting the following options from menu:
+    
+    Dataset: Population figures
+    Display type: Totals
+    Population type: All options selected
+    Years: 1951-2020
+    Country of origin: All countries
+    Country of destination: All countries
+    
+The second csv file was downloaded from the explorer
+after selecting the following options from menu: 
+
+    Dataset: Solutions
+    Display type: Totals
+    Solution type: All options selected
+    Years: 1951-2020
+    Country of origin/return: All countries
+    
+'''
```

Comment:
If we don't do this w should improves the download instructions. Is there a way to reference the Windows users Downloads directory instead of the Linux one?